src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -306,6 +307,59 @@ public async Task When_IsEnabled_Set()
 			Assert.AreEqual(contentPresenter.Foreground, foregroundColor);
 		}
 
+#if __ANDROID__
+		[TestMethod]
+		public async Task When_Text_IsWrapping_Set()
+		{
+			var textbox = new TextBox();
+
+			var initialHeight = 500;					
+			
+			textbox.Height = initialHeight;
+			
+			WindowHelper.WindowContent = textbox;
+
+			await WindowHelper.WaitForLoaded(textbox);
+
+			var actualHeigth = textbox.ActualHeight;
+
+			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
+			textbox.TextWrapping = TextWrapping.Wrap;
+			await WindowHelper.WaitForIdle();
+
+			var textBoxView = textbox.FindFirstChild<TextBoxView>();
+
+			Assert.IsTrue(textBoxView.LineCount > 1);
+		}
```

Comment:
Here's how you can change the test, by using the ActualHeight of the control. You need to place the control in a container which lets the TextBox set its own height (`StackPanel` in that case), so that when the wrapping is changed, its height changes.

The test will work on all platforms this way.

```suggestion
		public async Task When_Text_IsWrapping_Set()
		{
			var textbox = new TextBox();
			
			textbox.Width = 100;

			StackPanel panel = new() { Orientation = Orientation.Vertical };
			panel.Children.Add(textbox);

			WindowHelper.WindowContent = panel;

			await WindowHelper.WaitForLoaded(textbox);

			var originalActualHeigth = textbox.ActualHeight;

			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
			textbox.TextWrapping = TextWrapping.Wrap;
			await WindowHelper.WaitForIdle();

			textbox.ActualHeight.Should().BeGreaterThan(originalActualHeigth);
		}

```